### PR TITLE
HTML & CSS questions

### PR DESCRIPTION
**Pull request by Theweeksad**
added folder question-one and styles

### CODE REVIEW

File: CSS/Global-styles.css [hunk 1]
```diff
@@ -33,3 +33,17 @@ a {
...
     font-size: 2em;
 }
+
+form {
+    font-size: 16px;
+
+}
+
+form {
+    font-size: 16px;
```
[the remark this answers is not in the document]
Comment (Theweeksad) [reply CSS/Global-styles.css:43]: I have already exchanged for a relative unit (em).

File: CSS/Global-styles.css [hunk 2]
```diff
@@ -40,10 +40,14 @@ form {
...
+    font-size: 1.6em;
 }
 
 fieldset {
     margin: 0px;
     border: none;
 }
+
+legend {
+    font-size: 24px;
```
Comment (ripsamuel) [inline CSS/Global-styles.css:52]: pls use relative measurements

Comment (Theweeksad) [reply CSS/Global-styles.css:52]: I have already exchanged for a relative unit (em).

**Review by Brayan1425 — APPROVED** [review]

**Comment by ripsamuel** [comment]
please add  the `for` atribute to your inputs. to improve accessibility

**Review by ripsamuel — APPROVED** [review]

File: CSS/Homepage.css [hunk 3]
```diff
@@ -23,3 +30,43 @@
...
+.form__list {
+    display: flex;
+    flex-direction: column;
+    min-height: 50vh;
+    justify-content: space-between;
+}
+
+.title__question {
+    margin: 0px 35px 40px 35px;
+    ;
```
Comment (johanmendezb) [inline CSS/Homepage.css:44]: remo this semicolon

File: CSS/Homepage.css [hunk 4]
```diff
@@ -23,3 +30,43 @@
...
+}
+
+.form__item {
+    margin: 0px 35px 16px 45px;
```
Comment (johanmendezb) [inline CSS/Homepage.css:48]: right and left margins should be done at layout level

File: CSS/Homepage.css [hunk 5]
```diff
@@ -23,3 +30,43 @@
...
+/* question-one */
+.form__list {
+    display: flex;
+    flex-direction: column;
+    min-height: 50vh;
+    justify-content: space-between;
+}
+
+.title__question {
+    margin: 0px 35px 40px 35px;
```
Comment (johanmendezb) [inline CSS/Homepage.css:43]: right and left margins should be done at layout level

File: CSS/Homepage.css [hunk 6]
```diff
@@ -23,3 +30,43 @@
...
+.form__item {
+    margin: 0px 35px 16px 45px;
+}
+
+/*question-three*/
+.slider {
+    width: 80%;
+    height: 10px;
+    background: grey;
+    margin: 0px 39px 20px 50px;
```
Comment (johanmendezb) [inline CSS/Homepage.css:56]: right and left margins should be done at layout level

File: CSS/Homepage.css [hunk 7]
```diff
@@ -23,3 +30,43 @@
...
+    ;
+}
+
+.form__item {
+    margin: 0px 35px 16px 45px;
+}
+
+/*question-three*/
+.slider {
+    width: 80%;
```
Comment (johanmendezb) [inline CSS/Homepage.css:53]: avoid giving specific width to elements, if the margins are right you should not have this problem

File: CSS/Homepage.css [hunk 8]
```diff
@@ -23,3 +30,43 @@
...
+    width: 25px;
+    height: 25px;
+    border-radius: 15px;
+    background-color: black;
+}
+
+.slider__item {
+    display: flex;
+    justify-content: space-around;
+}
```
Comment (johanmendezb) [inline CSS/Homepage.css:72]: add a new line here

File: Flow/question_three.html [hunk 9]
```diff
@@ -4,10 +4,39 @@
...
 </head>
 <body>
+    <header class="header">
+        <h1 class="header__title">Title</h1>
+    </header>
+
     <main>
+        <form id="form__list" class="form__list">
+            <fieldset>
+                <legend class="title__question">¿Cual de estas variables nunca cambia el valor?</legend>
```
Comment (johanmendezb) [inline Flow/question_three.html:22]: remember that questions should be in english

File: index.html [hunk 10]
```diff
@@ -17,7 +19,7 @@ <h1 class="header__title">Title</h1>
       <h2 class="header__subtitle">Subtitle</h2>
     </header>
 
-    <main>
+    <main class="container">
       <section class="section">
         <p class="section__paragraph">
           Lorem ipsum dolor sit amet consectetur adipisicing elit. Provident
```
Comment (johanmendezb) [inline index.html:25]: please remove the lorem ipsum text

**Review by johanmendezb — CHANGES_REQUESTED** [review]
please review the comments plus make the questions in english

File: CSS/Homepage.css (hunk 4 above)
Comment (Theweeksad) [reply CSS/Homepage.css:48]: I already looked at the design and put the margins I had in the design

File: CSS/Homepage.css (hunk 6 above)
Comment (Theweeksad) [reply CSS/Homepage.css:56]: I already looked at the design and put the margins I had in the design

File: CSS/Homepage.css (hunk 7 above)
Comment (Theweeksad) [reply CSS/Homepage.css:53]: I fixed the problem by placing the range inside a parent so it will have a with="100%".

File: CSS/Homepage.css (hunk 8 above)
Comment (Theweeksad) [reply CSS/Homepage.css:72]: I have already put the new line

File: Flow/question_three.html (hunk 9 above)
Comment (Theweeksad) [reply Flow/question_three.html:22]: I have already changed each of the English questions.

File: index.html (hunk 10 above)
Comment (Theweeksad) [reply index.html:25]: I have already deleted the text lorem ipsum

File: CSS/Homepage.css [hunk 11]
```diff
@@ -23,3 +30,45 @@
...
+
+/*question-three*/
+.form__range {
+    margin: 0px 39px 20px 50px;
+}
+
+.slider {
+    width: 100%;
+    height: 10px;
+    background: grey;
```
Comment (ripsamuel) [inline CSS/Homepage.css:58]: this color and measure does not go in the design

**Review by ripsamuel — COMMENTED** [review]
please attacht screenshot in the
 PR

**Review by francoo-trainer — APPROVED** [review]

**Comment by francoo-trainer** [comment]
good job

File: CSS/Homepage.css (hunk 11 above)
Comment (Theweeksad) [reply CSS/Homepage.css:58]: hello, I have changed the color as it is in the design

**Pull request merged**